coffeecard/CoffeeCard.Library/Services/AccountService.cs

```
@@ -138,14 +138,23 @@ public async Task<User> RegisterAccountAsync(RegisterDto registerDto)
             return user;
         }
 
-        public bool VerifyRegistration(string token)
+        public async Task<bool> VerifyRegistration(string token)
         {
             Log.Information($"Trying to verify registration with token: {token}");
 
-            var user = VerifyTokenClaimAndUser(token);
+            User user;
+            try
+            {
+                user = await VerifyTokenClaimAndUserAsync(token);
+            }
+            catch (ApiException e)
+            {
+                // No log, since the ApiException should have been logged on throwing
+                return false;
+            }
```

Same question as earlier, why are we suppressing the exception?